Rubberduck.CodeAnalysis/Inspections/Concrete/ImplementedInterfaceMemberInspection.cs

```diff
@@ -43,7 +44,13 @@ public ImplementedInterfaceMemberInspection(Parsing.VBA.RubberduckParserState st
 
         protected override IEnumerable<IInspectionResult> DoGetInspectionResults()
         {
-            return State.DeclarationFinder.FindAllUserInterfaces()
+            var annotatedAsInterface = State.DeclarationFinder.Classes
+                .Where(cls => cls.Annotations.Any(an => an.Annotation.Name.Equals("Interface"))).Cast<ClassModuleDeclaration>();
```

Comment:
The predicate here should be `an.Annotation is InterfaceAnnotation`. Also I wonder why the Cast is necessary... `DeclarationFinder.Classes` should preerrably already return `ClassModuleDeclaration`s, but currently doesn't. That seems to be something of an oversight, though @MDoerner might correct me there...

Reply:
Yes - definitely a better predicate. Done.